site/content/docs/manifest/lb-web-service.ja.md

```diff
@@ -73,7 +73,64 @@ Service の名前。
 <a id="type" href="#type" class="field">`type`</a> <span class="type">String</span>  
 Service のアーキテクチャタイプ。 [Load Balanced Web Service](../concepts/services.ja.md#load-balanced-web-service) は、ロードバランサー及び AWS Fargate 上の Amazon ECS によって構成される、インターネットに公開するための Service です。
 
-{% include 'http-config.ja.md' %}
+<div class="separator"></div>
+
+<a id="http" href="#http" class="field">`http`</a> <span class="type">Boolean or Map</span>
+http セクションは Application Load Balancer と Serrice との連携に関するパラメータを含みます。
+
+Application Load Balancer を無効化する場合は、 `http: false` と指定します。 Load-Balanced Web Service では、Application Load Balancer または、Network Load Balancer が少なくとも 1 つ有効となっていなければならない事に注意してください。 
```

Comment:
typo: Load-Balanced Web Service -> Load Balanced Web Service

Reply:
Original text is ["Load-Balanced Web Service"](https://github.com/aws/copilot-cli/blob/64092562616ea3525db64020a5b79de400f03ec2/site/content/docs/manifest/lb-web-service.en.md?plain=1#L81).

But I think your suggestion is more correct. I will fix it.